spyder/plugins/completion/providers/kite/widgets/calltoaction.py

```diff
@@ -79,7 +79,9 @@ def __init__(self, textedit, ancestor):
         self.hide()
 
         is_kite_installed, __ = check_if_kite_installed()
-        if is_kite_installed:
+        installers_available = check_kite_installers_availability()
+
+        if is_kite_installed or not installers_available:
```

Comment:
@andfoy I think this will leave the `call_to_action` disable after the first time the installer gets unavailable but I'm unsure how to preserver the config in this case. It is ok to leave it disabled even if the installers get available later @ccordoba12 ?

Reply:
Also, I can manually go to the settings enable the `call_to_action` and get it again. Maybe a better place to add the validation is at `handle_processed_completions`: https://github.com/spyder-ide/spyder/blob/521839a573db348acf8bc61e5e00dd732db91573/spyder/plugins/completion/providers/kite/widgets/calltoaction.py#L96-L98

Something like:

```python
    installers_available = check_kite_installers_availability()
    if not installers_available:
        return
```